### PR TITLE
Merge playwright tests to Develop

### PR DESCRIPTION
**Pull request by Xen0Xys**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -1,39 +0,0 @@
...
-          curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh -s -- -y
-      - name: "Install dependencies"
-        run: |
-          npm install
-      - name: "Build Aladin Lite"
-        run: |
-          npm run build
-      - name: "Run some tests"
-        run: |
-          npm run test:build
```
Comment (bmatthieu3) [inline .github/workflows/test.yml:38]: I would not remove that. It runs some rust core tests on the core. There is only a few but it is still valuable to have them running on some CI.

File: package.json [hunk 2]
```diff
@@ -44,11 +44,14 @@
...
+    "test:update-snapshots": "npx playwright test --update-snapshots",
     "doc": "jsdoc -c jsdoc.json src/js src/js/shapes && cp aladin-logo.png docs/",
     "doc:dev": "npm run doc && open docs/index.html"
   },
   "devDependencies": {
+    "@playwright/test": "^1.45.2",
+    "@types/node": "^20.14.11",
     "happy-dom": "^10.11.0",
     "jsdoc": "^4.0.2",
     "vite": "^4.3.8",
```
Comment (ManonMarchand) [inline package.json:57]: I think you can remove vite-related dependencies

Comment (Xen0Xys) [reply package.json:57]: I think that vite is used for the dev server, but I will test to remove it.

Comment (ManonMarchand) [reply package.json:57]: You're right. But you might be able to remove the dependencies added in this commit https://github.com/cds-astro/aladin-lite/commit/402e27001581d287e7c6471a76c249148fce8dad

**Pull request merged**